themes/qgis-theme/forusers_download.html

```diff
@@ -35,61 +35,38 @@ <h4>{{ _('Download for Windows') }} <i class="icon-chevron-down pull-right"></i>
-                        <a href="https://download.osgeo.org/osgeo4w/v2/osgeo4w-setup.exe" class="reference external">
-                        <ul class="inline download-row">
-                          <li><i class="icon-large icon-download-alt"></i></li>
-                          <li class="osgeo-download"></li>
-                          <li class="download-text">{{ _('OSGeo4W Network Installer') }}</li>
-                        </ul>
-                        </a>
-                        <p>
-			{{ _('In the installer choose ') }}<strong>{{ _('Express Install') }}</strong> {{ _('and select') }} <strong>{{ _('QGIS') }}</strong> {{ _('to install the <em>latest release</em> or ') }} <strong> {{ _('QGIS LTR') }} </strong> {{ _('to install the <em>long term release</em>.') }}<br/>
-			{{ _('The express installations have several optional packages including non-free software. To avoid those you have to use the') }} <strong>{{ _('Advanced Install') }}</strong> {{ _('and choose') }} <strong> qgis </strong> {{ _('and/or') }} <strong> qgis-ltr </strong> {{ _('in the desktop section.') }}<br/> <br/>
-<!--
-			<strong><font color="red"><em>{{ _('CAUTION') }}:</em></font></strong> {{ _('Upgrades of old setups from OSGeo4W v1 using this repository are not supported. You need to do a fresh install or use a different directory.') }}<br/>
-			<strong><font color="red"><em>{{ _('CAUTION') }}:</em></font></strong> {{ _('32 bit binaries are not produced anymore.  Also Windows 7 no longer works as we are now using Python 3.9, which dropped support for it.') }}<br/>
--->
-                      <br/>
-                      </div>
-                      <div>
-                        <h5>{{ _('Standalone installers (MSI) from OSGeo4W packages (recommended for new users)') }}</h5>
-                        <h6>{{ _('Latest release (richest on features):') }}</h6>
-                        <a href="https://qgis.org/downloads/QGIS-OSGeo4W-{{ release|e }}-{{ binary|e }}.msi" class="reference external">
-                          <ul class="inline download-row">
-                              <li><i class="icon-download-alt icon-large"></i></li>
-                              <li class="{{ "qgis{}-download".format( "2" if version.startswith("2.") else "" ) }}"></li>
-                              <li class="download-text">{{ _('QGIS Standalone Installer Version %s')|format( version ) }}</li>
-                          </ul>
-                        </a>
-                        <a href="https://qgis.org/downloads/QGIS-OSGeo4W-{{ release|e }}-{{ binary|e }}.sha256sum" class="reference external button">
-                          <ul class="inline download-row">
-                              <li></li>
-                              <li></li>
-                              <li class="download-text">sha256</li>
-                          </ul>
+                        <a class="btn btn-large" href="https://qgis.org/downloads/QGIS-OSGeo4W-{{ release|e }}-{{ binary|e }}.msi">
```

Comment:
```suggestion
                        <a class="btn btn-large btn-success" href="https://qgis.org/downloads/QGIS-OSGeo4W-{{ release|e }}-{{ binary|e }}.msi">
```

+ change css for icon-large to a white logo (see https://qgis.org/en/_downloads/33f050ac04289004b33a4b296306d781/qgis-icon-white64.png)